path-animate/demo.html

```diff
@@ -117,7 +117,45 @@
                         .drawCircle(0, 0, Math.max(this.width, this.height) * 0.6);
                     highlight.updateCache();
                 }
-            }
+            };
+
+            airplane.setPosition = function () {
+                var temp = Math.floor(Math.random() * 4); // determines which side airplane will enter from
+                var endpoint = [];
+
+                if (temp < 2) { // airplane coming from either top or below
+                    airplane.x = Math.floor(Math.random() * canvas.width);
+                    endpoint[0] = Math.floor(Math.random()*canvas.width);
+
+                    if (temp === 0) { // airplane coming from top
+                        airplane.y = -airplane.height;
+                        endpoint[1] = canvas.height + airplane.height;
+
+                    } else if (temp === 1) { // airplane coming from below
```

Comment:
this else causes a redundant check, if it isn't 0, it can only be 1, so this just needs to be an else statement without the if.